src/modules/ekf2/EKF/ekf.h

```diff
@@ -281,7 +281,12 @@ class Ekf final : public EstimatorInterface
 	// return true if the local position estimate is valid
 	bool local_position_is_valid() const
 	{
-		return (!_horizontal_deadreckon_time_exceeded && !_control_status.flags.fake_pos);
+#if defined(CONFIG_EKF2_GNSS)
+		// if there's GPS module but disabled, we want to slack the fake pos check
+		return !_horizontal_deadreckon_time_exceeded && (!_control_status.flags.fake_pos || !_params.gnss_ctrl);
```

Comment:
Could you expand on the problem you needed to solve here?

If anything could we instead change the deadreckoning conditions contextually instead?

Also keep in mind the gnss_ctrl parameter isn't telling us if we have a module or not.

Reply:
This was something i added with a slightly different approach I tried. But you're correct its not necessary since the deadreckon time never exceeds while having fixed wing and gps disabled and therefore the fake gps does not get triggered.

This has the slight downside that we only have zero velocity / gyro updates but since we dont have any knowledge about local position this should be fine.